invenio_app_ils/schemas/documents/document-v1.0.0.json

```diff
@@ -160,19 +123,466 @@
+            "additionalProperties": false,
+            "properties": {
+              "material": {
+                "enum": [
+                  "addendum",
+                  "additional material",
+                  "data",
+                  "e-book",
+                  "e-proceedings",
+                  "editorial note",
+                  "erratum",
+                  "preprint",
+                  "publication",
+                  "reprint",
+                  "software",
+                  "translation"
+                ],
+                "minLength": 1,
+                "title": "Material to which the field refers",
+                "type": "string"
+              },
+              "schema": {
+                "description": "`HDL <http://handle.net>`_\n* `URN\n  <https://en.wikipedia.org/wiki/Uniform_Resource_Name>`_",
+                "enum": ["HDL", "URN"],
+                "minLength": 1,
+                "title": "Type of identifier",
+                "type": "string"
+              },
+              "source": {
+                "minLength": 1,
+                "type": "string"
+              },
+              "value": {
+                "minLength": 1,
+                "pattern": "^\\S+$",
+                "title": "Persistent identifier",
+                "type": "string"
+              }
+            },
+            "required": ["schema", "value"],
+            "type": "object"
+          },
+          "minItems": 1,
+          "title": "List of persistent identifiers",
+          "type": "array",
+          "uniqueItems": true
+        },
+        "external_system_identifiers": {
+          "items": {
+            "additionalProperties": false,
+            "properties": {
+              "schema": {
+                "enum": [
+                  "ARXIV",
+                  "ASIN",
+                  "DCL",
+                  "DESY",
+                  "DOE",
+                  "EBL",
+                  "FIZ",
+                  "HAL",
+                  "IEECONF",
+                  "INDICO.CERN.CH",
+                  "INIS",
+                  "INSPIRE",
+                  "INSPIRE-CNUM",
+                  "KEK",
+                  "LHCLHC",
+                  "SAFARI",
+                  "SCEM",
+                  "UDCCERN",
+                  "WAI01"
+                ],
+                "minLength": 1,
+                "pattern": "^\\w+$",
+                "type": "string"
+              },
+              "value": {
+                "minLength": 1,
+                "pattern": "^\\S+$",
+                "title": "External identifier",
+                "type": "string"
+              }
+            },
+            "required": ["schema", "value"],
+            "type": "object"
+          },
+          "minItems": 1,
+          "type": "array",
+          "uniqueItems": true
         }
-      }
+      },
+      "type": "object",
+      "title": "Document identifiers."
     },
-    "language": {
+    "languages": {
```

Comment:
Are there books in multiple languages in the same publication?

Reply:
there are standards and/or proceedings that can have multiple languages for one document